### PR TITLE
Fixing issue #4950

### PR DESCRIPTION
**Pull request by leonardlabat**
Use a deepmerge function to merge both globa level config & bidder specific config

## Type of change
- [X] Bugfix

## Description of change
Fixing issue https://github.com/prebid/Prebid.js/issues/4950

### CODE REVIEW

File: test/spec/utils_spec.js [hunk 1]
```diff
@@ -922,4 +922,34 @@ describe('Utils', function () {
...
+
+      const resultWithMergeDeep = utils.mergeDeep({}, object1, object2);
+      expect(resultWithMergeDeep).to.deep.equal({
+        propA: {
+          subPropA: 'abc',
+          subPropB: 'def'
+        }
+      });
+    });
+  });
```
Comment (snapwich) [inline test/spec/utils_spec.js]: I think you need some more tests dealing with other property types besides plain objects, also merging objects of different depths.

Comment (pm-harshad-mane) [reply test/spec/utils_spec.js]: @leonardlabat can you please add more test cases as suggested by @snapwich ?

Comment (leonardlabat) [reply test/spec/utils_spec.js]: Just added 2 test cases (and support for merging arrays)
Tell me if you think it still need more tests

**Comment by leonardlabat** [comment]
Hello guys,

Is there any blocking point with this PR ? 

Thanks !

**Pull request closed without merging**